compiler/damlc/tests/daml-test-files/ExceptionSemantics.EXPECTED.desugared-daml

```diff
@@ -9,7 +9,7 @@ data DA.Internal.Desugar.DamlException => E
   = E {}
   deriving (DA.Internal.Desugar.Eq, DA.Internal.Desugar.Show)
 instance DA.Internal.Desugar.HasMessage E where
-  message this@E = "E"
+  message this@E {} = "E"
```

Comment:
This is a consequence of https://github.com/digital-asset/ghc/pull/117/files#r866579065